support-frontend/assets/pages/contributions-landing/components/ContributionFormContainer.jsx

```diff
@@ -100,49 +101,61 @@ function withProps(props: PropTypes) {
     ...campaign || {},
   };
 
-  return props.paymentComplete ?
```

Comment:
this diff is quite confusing but essentially I've just replaced the ternary operator and added a new condition that checks `campaignCodeParameter`

Reply:
definitely prefer this for readability!